### DISCUSSION TITLE
Goreleaser nfpm bash/zsh/fish completion installation

### DISCUSSION THREAD

**Discussion by rfay** [goreleaser/goreleaser#3327] [Q&A] [answered]
In my package I'd like to properly install bash completions (and zsh and fish) and just realized that I'm not doing it properly. 

Config at https://github.com/drud/ddev/blob/badc85136b3cf586ebe2a59a9527cad776184f0d/.goreleaser.yml#L237-L253

The completions are being bundled in there because they're in the build, but they're just scripts and just dumped into /usr/bin on the target system (Ubuntu for example) and not properly installed.  For example, `/usr/bin/ddev_bash_completion.sh` is there just fine, but didn't get installed as bash completion.

What's the right way to configure the nFPM package to do bash/zsh/fish completions?

**Comment by caarlos0** [comment, marked as answer]
you can see an example in goreleaser config itself: https://github.com/goreleaser/goreleaser/blob/a1e681c9ba74054f07ea16a6279d2651b4b0a2f7/.goreleaser.yaml#L207-L252

basically, you'll need to use the `contents` section to map the files to their right places

**Reply by rfay** [reply]
Thanks, that should do it. I wish there were some kind of authoritative installer like homebrew does.

**Reply by caarlos0** [reply]
yeah, unfortunately, it may varies from distro to distro... those I'm using seem to be the "most accepted ones"

**Reply by rfay** [reply]
This seems to have sorted it for me, thanks for much.
